3.3/contrib/etc/generate_container_user

```diff
@@ -0,0 +1,8 @@
+# Set current user in nss_wrapper
+PASSWD_DIR="/opt/openshift/etc"
```

Comment:
@csrwng I moved `passwd` out of `./src` because that directory is used for the sources ;-) If you have 'passwd' file in your GIT repository then you can override the one we generate ;-)